changelog/10907.improvement.rst

```diff
@@ -0,0 +1,5 @@
+When an exception traceback to be displayed is completely filtered out (by mechanisms such as ``__tracebackhide__``, internal frames, and similar), now only the exception string and the following message are shown:
+
+"All traceback entries are hidden. Pass `--full-trace` to see hidden and internal frames.".
+
+Previously, the last frame of the traceback was shown, even though it was hidden.
```

Comment:
```suggestion
The last frame of the traceback is not mistakenly shown anymore when exceptions tracebacks are completely filtered out  (by mechanisms such as ``__tracebackhide__``, internal frames, and similar) without the `--full-trace` option.
```

Reply:
@Pierre-Sassoulas thanks for the review, the PR was on auto-merge so merged before I could reply.

I did want to explictly mention the complete message, for the benefit of users searching the string in search engines and such.

Reply:
Make sense ! No problem it was a nit anyway.